kolibri/core/assets/src/mixins/nav-components.js

```diff
@@ -0,0 +1,49 @@
+import {
+  isUserLoggedIn,
+  isSuperuser,
+  isAdmin,
+  isCoach,
+  isLearner,
+  canManageContent,
+} from 'kolibri.coreVue.vuex.getters';
+import { UserKinds } from 'kolibri.coreVue.vuex.constants';
+
+export default {
+  methods: {
+    filterByRole(navItem) {
+      if (!navItem.role) {
+        // No role defined, so always show
+        return true;
+      }
+      if (navItem.role === UserKinds.COACH) {
```

Comment:
Minor, but could we order these hierarchically?  Maybe use a switch statement?

Reply:
I've come to believe switch statements should be avoided in JS.

here's someone on the internet with a similar opinion: https://medium.com/chrisburgin/rewriting-javascript-replacing-the-switch-statement-cfff707cf045

Reply:
Missed this comment, but I think this is readable, clear, and gets the job done.

With the validation I am doing, we can also guarantee that one of these conditions will trigger.